src/components/bottom_bar/bottom_bar.xml

```diff
@@ -2,23 +2,27 @@
   <t t-name="o-spreadsheet-BottomBar">
     <div
       class="o-spreadsheet-bottom-bar o-two-columns d-flex align-items-center overflow-hidden"
+      t-att-class="{'ps-3 flex-fill': env.isSmall}"
       t-on-click="props.onClick"
       t-ref="bottomBar"
       t-on-contextmenu.prevent="">
-      <Ripple>
+      <Ripple class="env.isSmall ? 'order-2' : ''">
         <div
           class="o-sheet-item o-add-sheet me-2 p-1"
+          t-att-class="{'ms-2': env.isSmall}"
```

Comment:
seems a bit strange to have an `env.isSmall` that is used a bunch in the bottom bar, but never in the topbar. Maybe env.isSmall should just be defined in the bottom bar rather than on the whole spreadsheet then ?

Reply:
this will come later on with the mobile mode, don't worry :D